cmake/README.md

```diff
@@ -75,6 +75,7 @@ compile few of them using the options below (see [CMake Options](#cmake-options)
 * SCIP (`BUILD_SCIP`),<br>
   note: You can disable the support of SCIP solvers
   by using `-DUSE_SCIP=OFF` (`ON` by default).
+* GLPK (`BUILD_GLPK`)
```

Comment:
note: You can disable the support of GLPK solvers
  by using `-DUSE_GLPK=OFF` (`ON` by default).

Reply:
changed